### PR TITLE
Vitdet visualization

### PR DESCRIPTION
**Pull request by tuofeilunhifi**
support vitdet predictor and visualization

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ notebook
 * [self-supervised learning](docs/source/tutorials/EasyCV图像自监督训练-MAE.ipynb)
```
Comment (wenmengzhou) [inline README.md:64]: remerber also to update README_CN.md

File: easycv/predictors/detector.py [hunk 2]
```diff
@@ -252,6 +261,153 @@ def predict(self, input_data_list, batch_size=-1, to_numpy=True):
...
+        self.CLASSES = self.cfg.CLASSES
+        # dynamic adapt mmdet models
+        dynamic_adapt_for_mmlab(self.cfg)
+        self.model = build_model(self.cfg.model)
+        checkpoint = load_checkpoint(
+            self.model, checkpoint, map_location='cpu')
+        self.model.to(self.device)
+        self.model.eval()
+
+    def inference_detector(self, imgs):
```
Comment (wenmengzhou) [inline easycv/predictors/detector.py:294]: PredictorInterface use predict function to do inference

File: easycv/predictors/detector.py [hunk 3]
```diff
@@ -252,6 +261,153 @@ def predict(self, input_data_list, batch_size=-1, to_numpy=True):
         return output_list
 
 
+@PREDICTORS.register_module()
+class TorchViTDetPredictor(PredictorInterface):
+
+    def __init__(self, config, checkpoint):
```
Comment (wenmengzhou) [inline easycv/predictors/detector.py:267]: using exported model to do inference

File: README.md (hunk 1 above)
Comment (tuofeilunhifi) [reply README.md:64]: done

File: easycv/predictors/detector.py (hunk 2 above)
Comment (tuofeilunhifi) [reply easycv/predictors/detector.py]: done

File: easycv/predictors/detector.py (hunk 3 above)
Comment (tuofeilunhifi) [reply easycv/predictors/detector.py:267]: done

File: easycv/predictors/detector.py [hunk 4]
```diff
@@ -252,6 +261,162 @@ def predict(self, input_data_list, batch_size=-1, to_numpy=True):
...
+        checkpoint = load_checkpoint(
+            self.model, checkpoint, map_location='cpu')
+        self.model.to(self.device)
+        self.model.eval()
+
+    def predict(self, imgs):
+        """Inference image(s) with the detector.
+
+        Args:
+            model (nn.Module): The loaded detector.
```
Comment (wenmengzhou) [inline easycv/predictors/detector.py:298]: update doc str

**Pull request closed without merging**